Package.swift

```diff
@@ -0,0 +1,48 @@
+// swift-tools-version:5.3.0
+
+import PackageDescription
+
+let package = Package(
+    name: "ably-cocoa",
+    platforms: [
+        .macOS(.v10_11),
+        .iOS(.v9),
+        .tvOS(.v10)
+    ],
+    products: [
+        .library(
+            name: "ably-cocoa",
+            targets: ["Ably"]
+        ),
+    ],
+    dependencies: [
+        .package(name: "msgpack", url: "https://github.com/rvi/msgpack-objective-C", from: "0.4.0"),
+        .package(name: "AblyDeltaCodec", url: "https://github.com/ably/delta-codec-cocoa", .branch("main"))
+    ],
+    targets: [
+        .target(
+            name: "SocketRocket",
+            path: "SocketRocket/SocketRocket",
+            publicHeadersPath: "include",
+            cSettings: [
+                .headerSearchPath("PrivateHeaders"),
+                .headerSearchPath("include")
+            ]
+        ),
+        .target(
+            name: "Ably",
+            dependencies: [
+                "SocketRocket",
+                .byName(name: "msgpack"),
+                .byName(name: "AblyDeltaCodec")
+            ],
+            path: "Source",
+            publicHeadersPath: "include",
+            cSettings: [
+                .headerSearchPath("PrivateHeaders/**"),
```

Comment:
Same comment for the `PrivateHeaders/` in SocketRocket.

Reply:
I messaged with @lukasz-szyszkowski  over slack. This was required, because Swift Packages seem to look for the `Ably/` directory within the header search path. This is because the imports are using `#import "Ably/HeaderName.h"`. There were 2 options to fix: change the imports so it doesn't have `"Ably/`, or add symlinks to `PrivateHeaders/ably/` directory. We went with the second option.